Breeding-Insight/bi-web
# BI-1647 (View Experiments: Observation Dataset Table)

# Description
[BI-1647](https://breedinginsight.atlassian.net/browse/BI-1647) (View Experiments: Observation Dataset Table)

# Dependencies
bi-api: branch _feature/BI-1647_

# Testing
1. Import an Experiment with multiple Observation Units and multiple observations per Observation Unit.
2. From the **Experiments & Observations** page, navigate to the imported experiment (from step 1).
3. Verify that the data displayed in the table is correct.
4. Test that the sort and filter feature works on each column of the table.

# Checklist:
